kafka/kafka-common/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -409,6 +418,7 @@ public void onComplete(Map<TopicPartition, OffsetAndMetadata> map, Exception e)
       logger.debug("Commit offsets complete {} ", Joiner.on(';').withKeyValueSeparator("=").join(map));
```

Comment:
Not directly related to your change, Having @Override keyword for onComplete helps.